src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -12,69 +12,6 @@
         The initial schema has the '00000000000001' id, so that it is over-written if we re-generate it.
     -->
     <changeSet id="00000000000001" author="jhipster">
-        <createTable tableName="jhi_user">
```

Comment:
`createTable` is not supported by the Neo4j extension

Reply:
Maybe the extension could throw.

Reply:
Nevermind, I doubt many people will try to execute a sql liquibase in neo4j. 😄